src/SprykerEco/data/translation/Zed/de_DE.csv

```diff
@@ -0,0 +1,3 @@
+"Author: A user who can create data sources, datasets, analyses, and dashboards. <br/>Reader: A user who has read-only access to dashboards.","Autor: Ein Benutzer, der Datenquellen, Datensätze, Analysen und Dashboards erstellen kann. <br/>Leser: Ein Benutzer, der nur Lesezugriff auf Dashboards hat."
```

Comment:
please make sure this message is really translated in this format

Reply:
Checked, it works with `raw` filter.